matrix/common/dynamo_handler.py

```diff
@@ -203,3 +214,36 @@ def _increment_field(self, table, key_dict: dict, field_enum: TableField, increm
             time.sleep(.5)
 
         return start_value, new_value
+
+    def get_request_hash(self, request_id):
```

Comment:
To minimize repeated code, we've been using `get_table_item` to read items from tables. Then, I'd argue `get_request_hash` would live in `RequestTracker`

Reply:
Oh I see now - instead of using `RequestTracker`, I would update `RequestCache.retrieve_hash` to use `get_table_item`